backend/api/appsettings.Staging.json

```diff
@@ -0,0 +1,41 @@
+{
+  "AzureAd": {
+    "ClientId": "ea4c7b92-47b3-45fb-bd25-a8070f0c495c",
+    "ClientSecret": "Fill in ASP.NET Secret Manager"
+  },
+  "KeyVault": {
+    "VaultUri": "https://PlaceboFlotillaDevKv.vault.azure.net/"
+  },
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Information",
+      "Microsoft.Hosting.Lifetime": "Information"
+    }
+  },
+  "AllowedHosts": "*",
+  "AllowedOrigins": [
+    "https://*.equinor.com/",
+    "http://localhost:3001",
+    "https://localhost:3001"
+  ],
+  "Mqtt": {
+    "Host": "localhost",
+    "Port": 1883,
+    "Username": "flotilla",
+    "Topics": [
+      "isar/+/robot",
+      "isar/+/mission",
+      "isar/+/task",
+      "isar/+/step",
+      "isar/+/battery",
+      "isar/+/pose"
+    ],
+    "MaxRetryAttempts": 15,
+    "ShouldFailOnMaxRetries": true
+  },
+  "Database": {
+    "UseInMemoryDatabase": false,
+    "ConnectionString": ""

```

Comment:
Should this be here?

Reply:
Yes, it's retrieved from the keyvault. It's collected from keyvault when using the staging environment as that will use the azure database.

Reply:
But if it's retrieved from keyvault it's not set in this file so maybe having it there is misleading as we really should never fill in the value?

Reply:
My understanding was that having it here is what caused it to be retrieved automatically from the keyvault but maybe I've misunderstood the functionality?

Reply:
Ah, no, it doesn't have to be there, the keyvault is loaded during startup and when we retrieve values later it simply looks in the keyvault as one of the lookup sources, it doesn't have to be defined here